tests/unit/backend/data/test_LocalDataService.py

```diff
@@ -1825,21 +1842,19 @@ def test_readWriteNormalizationState():
 
 def test_readDetectorState():
     localDataService = LocalDataService()
-    localDataService._readPVFile = mock.Mock()
+    testDetectorState = {
```

Comment:
testDetectorState = mockReadDetectorState("67890")  # It's not the same, but it shouldn't matter...